### PR TITLE
Reaper responds to file:ingest:end from odo and processes geojson ingest

### PR DESCRIPTION
**Pull request by rajones-hntb**

### CODE REVIEW

File: apps/reaper/lib/reaper/init.ex [hunk 1]
```diff
@@ -17,6 +17,7 @@ defmodule Reaper.Init do
     |> Enum.each(fn dataset ->
       Logger.debug("Migrating extraction dataset #{dataset.id}")
       Reaper.Horde.Supervisor.start_data_extract(dataset)
+      Process.sleep(250)
```
Comment (ManApart) [inline apps/reaper/lib/reaper/init.ex]: @JarredOlson Is this sleep still needed?

File: apps/reaper/lib/reaper/decoder/geojson.ex [hunk 2]
```diff
@@ -28,7 +28,8 @@ defmodule Reaper.Decoder.GeoJson do
 
   @impl Reaper.Decoder
   def handle?(source_format) when is_binary(source_format) do
-    String.downcase(source_format) == "application/geo+json"
+    String.downcase(source_format) == "application/geo+json" ||
+      String.downcase(source_format) == "application/zip"
```
Comment (JarredOlson) [inline apps/reaper/lib/reaper/decoder/geojson.ex]: Do we need this here? Odo should be emitting a hosted file with the source format of `application/geo+json`  if it isn't I think we should change Odo and not do this here.

File: apps/reaper/lib/reaper/init.ex (hunk 1 above)
Comment (ManApart) [reply apps/reaper/lib/reaper/init.ex]: Gonna keep the sleep.

File: apps/reaper/lib/reaper/decoder/geojson.ex (hunk 2 above)
Comment (rajones-hntb) [reply apps/reaper/lib/reaper/decoder/geojson.ex]: Removed this code

**Pull request merged**